mlflow/utils/databricks_utils.py

```diff
@@ -731,3 +733,65 @@ def get_databricks_env_vars(tracking_uri):
         env_vars.update(workspace_info.to_environment())
 
     return env_vars
+
+
+def _init_databricks_cli_config_provider(entry_point):
```

Comment:
This function is from `spark/driver/python/python_shell/dbruntime/databricksCli.py`

once legacy databricks-cli is removed from DBR, the function in DBR doesn't work any more.

So I inline it into MLflow with proper changes.